examples/hybrid/sphere/balanced_flow_rhoe.jl

```diff
@@ -1,25 +1,20 @@
 using Test
 using ClimaCorePlots, Plots
-using DiffEqCallbacks
-using JLD2
 
 include("baroclinic_wave_utilities.jl")
 
-jld2_callback = PeriodicCallback(output_writer, 864000; initial_affect = true)
+const sponge = false
 
-driver_values(FT) = (;
-    zmax = FT(30.0e3),
-    velem = 10,
-    helem = 4,
-    npoly = 4,
-    tmax = FT(60 * 60),
-    dt = FT(5.0),
-    ode_algorithm = OrdinaryDiffEq.SSPRK33,
-    jacobian_flags = (; ∂𝔼ₜ∂𝕄_mode = :no_∂P∂K, ∂𝕄ₜ∂ρ_mode = :exact),
-    max_newton_iters = 2,
-    save_every_n_steps = 10,
-    additional_solver_kwargs = (;), # callback = jld2_callback), 
+space = ExtrudedSpace(;
+    zmax = FT(30e3),
+    zelem = 10,
+    hspace = CubedSphere(; radius = R, pelem = 4, npoly = 4),
 )
+tend = FT(60 * 60)
+dt = FT(5)
+dt_save_to_sol = FT(50)
+# dt_save_to_disk = FT(60 * 60 * 24 * 10)
```

Comment:
If I understand correctly, `dt_save_to_disk = FT(0)` means no jld2 output? Then, can we add a `const dt_save_to_disk = FT(0)` above together with the `const sponge` and use `dt_save_to_disk=dt_save_to_disk` here.

Reply:
`dt_save_to_disk = FT(0)` is already specified as the default in `driver.jl`. I also did not want to make it a const since is is not used in any functions and is subject to change during testing.

Reply:
The commented-out line was only included to match your comment `# callback = jld2_callback),`

Reply:
How about we move it in the experiment files so that people who want to save jld2's know where to change. I was doing the comment only because I did not figure out a simple way without the comment.